pisces/stations.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
+from sqlalchemy import func, or_
+from obspy.core import UTCDateTime, Stream
+import obspy.geodetics as geod
+
+from pisces.io.trace import wfdisc2trace
+from pisces.util import make_wildcard_list, _get_entities
+
+import warnings
+
+def filter_network(query, net=None, netname=None, auth=None, sta=None,  time_=None, **tables):
+    """ipython
+    Parameters
```

Comment:
Please remove unused parameters and spell out descriptions for all docstrings.

Reply:
yeah...definitely going to fix docstrings before merge

Reply:
Filled out docstrings